### ISSUE TITLE
Species missing from timecourse result

### ISSUE THREAD

**Issue by matthiaskoenig** [copasi/basico#17]
I ran a simple timecourse simulation with `copasi-basico==0.16`. I have a species S1 in the model which does not appear in the results data frame. Model and code attached.
[tellurium_sim.zip](https://github.com/copasi/basico/files/8417242/tellurium_sim.zip)

Expected behavior: columns: `S1, S2, Values[k1]`
Current behavior: columns `S2, Values[k1]`

```
from basico import *

basico.load_model("tellurium_sim.sbml")

print("--- c1 ---")
tc = run_time_course(0, 5, 5)
print(tc)

print("--- c2 ---")
set_species(name='S1', initial_concentration=2)
tc = run_time_course(0, 5, 5)
print(tc)

```
results in 
```
--- c1 ---
       S2  Values[k1]
Time                 
0.0   0.0         0.0
1.0   0.0         0.0
2.0   0.0         0.0
3.0   0.0         0.0
4.0   0.0         0.0
5.0   0.0         0.0
--- c2 ---
       S2  Values[k1]
Time                 
0.0   0.0         1.0
1.0   1.0         1.0
2.0   2.0         1.0
3.0   3.0         1.0
4.0   4.0         1.0
5.0   5.0         1.0
```
There is a species `S1` in the model, but somehow it is missing from the results DataFrame? Perhaps indexing issue or similar thing.

**Comment by fbergmann** [comment]
The reason S1 is not in the output, is because s1 is not changing during the simulation at all (its not part of any modification, and so does not have to be calculated). IT would be the same in the COPASI user interface, where by default only the values that change would be part of the default output. 

So this does work as intended here. Now that i fixed the bug #18, this should be fine, rigth?

**Comment by matthiaskoenig** [comment]
Thanks for the explanation, solves my issue. Was not obvious for me why S1 was not part of the results.
